perf/lua/box_select.lua

```diff
@@ -1,27 +1,34 @@
 --
```

Comment:
commit message to "perf: standardize an output of box_select benchmark "

Reply:
In that case, the commit title is more than 50 symbols, so I shorten it to this one.